android/core/src/main/java/com/mongodb/stitch/android/core/internal/StitchAppClientImpl.java

```diff
@@ -262,6 +275,41 @@ public boolean isLoggedIn() {
     return getAuth().isLoggedIn();
   }
 
+  @Nullable
+  @Override
+  public String getActiveUserId() {
```

Comment:
[q] what is the advantage of this over just doing `getUser().getId()`?

Reply:
Compartmentalization. No need to expose the developer to fields they do not need.

Reply:
I mean the developer might use the other user fields, but I guess now that I think about it, this can be pretty convenient, so it's fine to leave as is